crates/router/src/connector/opennode/transformers.rs

```diff
@@ -114,7 +114,7 @@ impl<F, T>
                 mandate_reference: None,
                 connector_metadata: None,
                 network_txn_id: None,
-                connector_response_reference_id: None,
+                connector_response_reference_id: Some(item.response.data.id),
```

Comment:
Hello @eigengravy,

The `connector_response_reference_id` should be mapped to the merchant reference ID of the connector response, which is `order_id` according to the API documentation of OpenNode. Please add an optional field called `order_id` in the payment response and use it here.

Thank you.

Reply:
Please refer api documentation of OpenNode [here](https://developers.opennode.com/reference/create-charge)

Reply:
Made the required changes.